src/app/templates/server/middleware/logger.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
+// @slajax TODO: re-write this logger so it scales
 const winston = require('winston');
 
-module.exports = function(options = {}) {
-  return function(error, req, res, next) {
+module.exports = function (options = {}) {
+  return function (error, req, res, next) {
     if (error) {
-      const message = `${error.code ? `(${error.code}) ` : '' }Route: ${req.url} - ${error.message}`;
-      
+      const message = `${error.code ? `(${error.code}) ` : ''} Route: ${req.url} - ${error.message}`;
+
       if (error.code === 404) {
-        winston.info(message);
-      }
-      else {
-        winston.error(message);
-        winston.info(error.stack);
+        if (process.env.NODE_ENV !== 'testing') {
+          winston.info(message);
+        }
+      } else {
+        if (process.env.NODE_ENV !== 'testing') {
+          winston.error(message);
+          winston.info(error.stack);
+        }
       }
```

Comment:
I've got a few ideas for how we can improve this to support real time logging once we have an Elastic Search adapter. Winston is fine for now, we mostly just need to support more consistency with how it's being used vs debug.